front/src/modules/timesStops/hooks/useTimeStopsColumns.tsx

```diff
@@ -167,12 +169,34 @@ export const useTimeStopsColumns = <T extends TimeStopsRow>(
             alignRight: true,
           })
         ),
+        ...(isOutputTable && {
+          component: ({ rowData }) => {
+            const [digits, unit] = rowData.theoreticalMargin?.split(NO_BREAK_SPACE) ?? [];
```

Comment:
Can you add an early return if `theoreticalMargin` is undefined ? You can return null

Reply:
Sure!